asio/CMakeLists.txt

```diff
@@ -0,0 +1,240 @@
+# Minimum supported CMake version
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
```

Comment:
`FATAL_ERROR` has been a no-op for the longest time. Please read the docs:  
https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html

Reply:
I will change it to `3.13...3.20`